src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,32 @@
-export const TodoFilter = () => (
+import React from 'react';
+import { FilteringOptions } from '../../types/FilteringOptions';
+
+type Props = {
+  setFilteringOption: (option: FilteringOptions) => void,
+  filteringOption: string,
+  setSearchQuery: (query: string) => void,
+  searchQuery: string,
+};
+
+export const TodoFilter: React.FC<Props> = (
+  {
```

Comment:
It is better to destructure the Props directly in the function parameters for better readability.